components/automate-ui/src/app/page-components/profile/profile.component.scss

```diff
@@ -98,3 +98,8 @@ a, button:not(.dropdown-toggle) {
   right: 16px;
   top: 8px;
 }
+
+.version:after {
+  content: '\A';
+  white-space: pre;
+}
```

Comment:
I'm no CSS expert but it seems kinda non-obvious what's going on here. I had to look up what `content: '\A'` does to see why that's here. :)

This might be less prone to breaking in the future with a separately styled `<ul>` instead.